.github/workflows/celerity_ci.yml

```diff
@@ -37,7 +54,7 @@ jobs:
     if: github.event.pull_request
     runs-on: [self-hosted, nvidia]
     env:
-      container-workspace: /__w/${{ github.event.repository.name }}/${{ github.event.repository.name }}
+      container-workspace: <placeholder>
       build-dir: /root/build
     container:
```

Comment:
This container should also point to `:latest`.

Reply:
Fixed!